### DISCUSSION TITLE
FFmpeg 6 works wih Appmoverscript 3.0.59

### DISCUSSION THREAD

**Discussion by eineb01** [007revad/Synology_app_mover#102] [Show and tell]
Hi Dave,
FFmpeg 6 also works with AppmoverScript 3.0.59 and later. But FFmpeg 6 has not yet been added to the table.

Which package is meant by AVR Center? I haven't heard of it either.

**Comment by 007revad** [comment]
Thanks. I've added FFmpeg 6 to the table.

I can't find AVR Center anywhere online. I've deleted it now.

While getting the icons for all the packages I ended up with 2 different icons for AVR Logger. The current one and a older looking one that may have been for a DSM 5 package. For each new package I added I copied and pasted the Log Center line and then edited it. I may have intended to to change the old one to AvrLogger v2 or something but then forgot and accidentally left it as AvrCenter.
